app/services/validation_beta_service.rb

```diff
@@ -30,6 +30,17 @@ def tell_induction_coordinators_to_check_ect_and_mentor_information
     end
   end
 
+  def tell_induction_coordinators_we_asked_ects_and_mentors_for_information
+    InductionCoordinatorProfile.find_each do |ic|
+      ic.schools.not_opted_out.each do |school|
+        if chosen_programme_and_not_in_beta(school)
+          send_asked_ects_and_mentors_for_information(ic, school)
+          break
+        end
+      end
+    end
+  end
```

Comment:
Will this send at most one email per IC? (due to `break`)

If so, is it a good idea to include school name? 

```
  def tell_induction_coordinators_to_check_ect_and_mentor_information
    InductionCoordinatorProfile.find_each do |ic|
      if ic.schools.not_opted_out.any? { |school| chosen_programme_and_not_in_beta(school) }
        send_check_ect_and_mentor(ic)
      end
    end
  end
```
seems to be doing something very similar, and because it does not need school name, it is more functional.

Also, just in general, if we include the school name in the email, should we not send one email per school? If an IC has two schools, and they get one email with one school name, will they get confused and assume the other school is irrelevant?

Reply:
I agree — I'll return to Shani about this.

Reply:
My code intention was to send to one per IC because multiple didn't make sense to me. We are essentially just asking them to sign in. I'll confirm and update if neccessary.